packages/venia-ui/lib/components/FilterModal/filterFooter.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { bool, func, shape, string } from 'prop-types';
+
+import { mergeClasses } from '../../classify';
+import Button from '../Button';
+import defaultClasses from './filterFooter.css';
+
+const FilterFooter = props => {
+    const { applyFilters, hasFilters, resetFilters } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    return (
+        <div className={classes.root}>
+            <Button disabled={!hasFilters} onClick={resetFilters}>
+                {'Reset Filters'}
+            </Button>
+            <Button
+                disabled={!hasFilters}
```

Comment:
If I toggle a filter and click `Apply Filters`, but then want to clear it (by toggling it again or clicking the `x`) without clicking "reset", how do I do that? Right now if you cannot "apply" an empty set of filters but I would assume it would allow it. This behavior is present on develop so we don't have to change it but it jumped out as a weird UX when I did it.